### ISSUE TITLE
Question about Wazuh Manager (Server) ossec.conf configuration with both Ubuntu and Windows endpoints. 

### ISSUE THREAD

**Issue by WhiskeySheppard** [wazuh/wazuh#25434]
|Wazuh version|Component|Install type|Install method|Platform|

Wazuh v4.8.2  The other requested info isn't relevant to my issue

| X.Y.Z-rev | Wazuh component | Manager/Agent | Packages/Sources | OS version |

Wazuh management is running on Ubuntu 24.04.1 LTS 

I'm just requesting clarification on setup for the server ossec.conf with multiple OS endpoints reguarding the VirusTotal integration. 

Following through the guide, "https://documentation.wazuh.com/current/proof-of-concept-guide/detect-remove-malware-virustotal.html" It goes over the config settings for Ubuntu endpoint, Which I got to work and the threat emulation works as expected. when adding: 

![image](https://github.com/user-attachments/assets/5bdbf02d-a115-4b47-acd0-729089a40aa9)

To the server ossec.conf. 
And everything works on my ubuntu endpoint. But trying to get it to work on my Windows endpoints also, I wasn't sure how to proceed. Because the guide says to add: 

![image](https://github.com/user-attachments/assets/537c536d-fe52-4c70-8704-863b86ac41bc)

To the server ossec.conf for Windows endpoint. But I wasn't sure if it needed to look like this: 

![image](https://github.com/user-attachments/assets/58582ff8-fbe2-41bd-b1c7-7b554304936c)

when trying to use the virustotal integration to work for both Windows and Ubuntu endpoints. Since they're pretty much the same other than one is directing it to executable: >remove-threat.sh<, and the other is directing it to executable: >remove-threat.exe<.  Or if I'm supposed to combine the two types of executables something like this:

![image](https://github.com/user-attachments/assets/817509a4-6873-4402-8a7a-5a269cc9667f)

with a coma, or something between the two types of executables? 

In short, I'm just looking for a ossec.conf example when using virustotal intigration for both Windows and Ubuntu endpoints.

**Comment by davidcr01** [comment]
Hello.

Yes, you are fine with that configuration. The correct configuration would be the following, appending the Windows endpoint configuration and the Ubuntu endpoint configuration:

```conf
<ossec_config>
  <command>
    <name>remove-threat</name>
    <executable>remove-threat.exe</executable>
    <timeout_allowed>no</timeout_allowed>
  </command>

  <command>
    <name>remove-threat</name>
    <executable>remove-threat.sh</executable>
    <timeout_allowed>no</timeout_allowed>
  </command>

  <active-response>
    <disabled>no</disabled>
    <command>remove-threat</command>
    <location>local</location>
    <rules_id>87105</rules_id>
  </active-response>
</ossec_config>
```

Hope it helps
